pykokkos/linalg/l3_blas.py

```diff
@@ -0,0 +1,53 @@
+import pykokkos as pk
+
+# Level 3 BLAS functions
+
+def dgemm(alpha: float,
+          view_a,
+          view_b,
+          beta: float = 1.0):
```

Comment:
This should probably default to `0`, and we should add the `view_c` (optional output) argument per my misunderstanding of the usual interface in https://github.com/scipy/scipy/issues/17594.